acq4/devices/Pipette/pipette.py

```diff
@@ -198,10 +198,14 @@ def checkRangeOfMotion(self, pos, tolerance=500e-6):
         if bad_axes:
             axis_names = {0: 'x', 1: 'y', 2: 'z'}
             axes = ', '.join(axis_names[axis] for axis in bad_axes)
+            pos = self.mapGlobalToParent(pos)
```

Comment:
I don't think mapGlobalToParent gets you the coordinates you want here. Try following the position coordinate forward until it is actually sent to the stage device so you can see the transforms applied. I think it will involve Piptte._solveGlobalStagePosition and Stage.mapGlobalToDevicePosition.

Reply:
uh oh! that's what I'm using the _check_ the boundaries, too. okay, yeah, for a _manipulator_ to get to `pos` would be silly we need "when the pipette is at pos, what would its manipulator's position be?".